Mapify/Utils/Extensions.cs

```diff
@@ -17,6 +17,8 @@ namespace Mapify.Utils
 {
     public static class Extensions
     {
+        private const string jObjectName = "mapify";
```

Comment:
const's should use SCREAKING_SNAKE_CASE. It could be named something like `SAVE_KEY_NAME`

Reply:
changed